Revert "Revert "Tool list package id feature doc update"" (#32003)
dotnet/docs: docs/core/tools/global-tools.md

As shown in the preceding example, the list shows local tools. To see global tools, use the `--global` option. To see tool-path tools, use the `--tool-path` option.

To list a specific tool, use the [dotnet tool list <PACKAGE_ID>](dotnet-tool-list.md) command:

```dotnetcli
dotnet tool list dotnetsay
```

> The output will be only contain information of that tool if it is installed, similar to the following example:

Reviewer: ```suggestion
The output will only list that tool if it's installed, similar to the following example:
```
